api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java

```diff
@@ -594,7 +595,8 @@ public static final class EncounterTemplates {
             Apps.NOTES_SUMMARY,
             Apps.COVID_LAB_RESULTS,
             Apps.RECENT_LAB_RESULTS,
-            Apps.ALL_LAB_RESULTS
+            Apps.ALL_LAB_RESULTS,
+            Apps.BIOMETRICS_WIDGET
```

Comment:
If we aren't adding this to the main clinicial dashboard we don't need to add this here.

Reply:
Ah, but maybe this is a correction because it already is on the main clinician dashboard and we never provided an order for it?

Reply:
Good point @mogoodrich .  The ticket description was to add to the main clinical dashboard but then it became clear that this was for the HIV dashboard only.  I will remove from this line.  Thanks.